cypress/integration/intro.spec.ts

```diff
@@ -4,15 +4,22 @@ describe('Intro', () => {
     cy.visit('/')
   })
 
+  describe('Intro page fetch', () => {
+    it('fetches the intro page content', () => {
+      cy.get('.markdown-body>div.alert-danger>p')
+      .contains('test content')
+    })
+  })
+
   describe('Cover Button are hidden when logged in', () => {
     it('Sign in Cover Button', () => {
       cy.get('.cover-button.btn-success')
-        .should('not.exist')
+      .should('not.exist')
```

Comment:
any reasons why you remove the indentation here, @mrdrogdrog ?

Reply:
it guess that was eslint

Reply:
done